packages/client/src/__tests__/integration/happy/exhaustive-schema-mongo/__snapshots__/test.ts.snap

```diff
@@ -90,31 +91,46 @@ Prisma.NullTypes = {
 // https://github.com/microsoft/TypeScript/issues/3192#issuecomment-261720275
 function makeEnum(x) { return x; }
 
-exports.Prisma.PostScalarFieldEnum = makeEnum({
+exports.Prisma.AScalarFieldEnum = makeEnum({
```

Comment:
This feels hard to review and error-prone, is it because of changes of in ordering in the dmmf?

Reply:
If so, I wonder if it could help to "sort" the dmmf before hand.

Reply:
That's actually exactly what engine started doing in this PR - previously, order of enums in DMMF was consistent, albeit arbitrary, now `TransactionIsolationLevel` got added at arbitrary point, which was not consistent between the runs. To fix this, engine started sorting enums by name alphabetically. So, this diff is bigger, but we should be good for the future.